utilities/blob_db/blob_db_impl.cc

```diff
@@ -1042,6 +1036,18 @@ Status BlobDBImpl::PutWithTTL(const WriteOptions& options,
           : -1);
 }
 
+Slice BlobDBImpl::GetCompressedSlice(const Slice& raw,
+                                     std::string* compression_output) const {
```

Comment:
I believe you just overlooked this issue while trying to make changes rapidly 😃 
No need to pass in compression_output as you are returning it; just declare it locally in this function. Also lines 925 and 1061 can be removed.